sdk/eventhub/tests-mgmt.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      AllocateResourceGroup: false
+      ServiceDirectory: eventhub
+      BuildTargetingString: azure-mgmt-eventhub
+      MatrixFilters:
+        - PythonVersion=^(?!pypy3).*
```

Comment:
is this for filtering out pypy3?
I guess it's not needed for the mgmt libs as they support pypy3.

(uamqp doesn't support pypy3 so we ignore pypy3 test in sb and eh)